x-pack/test/functional/apps/ml/settings/filter_list_creation.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import { asyncForEach } from './common';
+
+export default function ({ getService }: FtrProviderContext) {
+  const ml = getService('ml');
+  const testDataList = [1, 2, 3].map((n) => ({
+    filterId: `test_create_filter_${n}`,
+    description: `test description ${n}`,
+    keywords: ['filter word 1', 'filter word 2', 'filter word 3'],
+  }));
+
+  describe('filter list creation', function () {
+    before(async () => {
+      await ml.testResources.setKibanaTimeZoneToUTC();
+      await ml.securityUI.loginAsMlPowerUser();
+    });
+
+    after(async () => {
+      await ml.api.cleanMlIndices();
+      // clean up created filters
+      await asyncForEach(testDataList, async ({ filterId }) => {
+        await ml.api.deleteFilter(filterId);
+      });
+    });
+
+    it('filter list creation creates new filter list', async () => {
```

Comment:
See above, I'd suggest to make this `creates new filter list`.

Reply:
Updated here 20c6b2d